generator/src/test/java/com/scottlogic/deg/generator/cucumber/longerThan.feature

```diff
@@ -0,0 +1,671 @@
+    Given foo is longer than 1
+    And foo is anything but less than or equal to 10
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+      | 2      |
+      | 12      |
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 12    |
+
+  Scenario: Running a 'longerThan' request alongside a granularTo constraint should be successful
+    Given foo is longer than 1
+    And foo is granular to 0.1
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 0.1      |
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 0.1   |
+
+  Scenario: Running a 'longerThan' request alongside a granularTo constraint should be successful
+    Given foo is longer than 1
+    And foo is anything but granular to 1
+    And foo is in set:
+      | "a"     |
+      | "aa"    |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+      | 2      |
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "aaaaaaaaaaa" |
+      | "aaaaaaaaaaaaa" |
+      | 1      |
+      | 2      |
+
+  Scenario: Running an 'longerThan' request alongside a after constraint should be successful
+    Given foo is longer than 1
+    And foo is after 2019-01-01T00:00:00.000
+    And foo is in set:
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.001" |
+      |  2019-01-01T00:00:00.001   |
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.001" |
+      |  2019-01-01T00:00:00.001   |
+
+
+  Scenario: Running an 'longerThan' request alongside a after constraint should be successful
+    Given foo is longer than 1
+    And foo is anything but after 2019-01-01T00:00:00.000
+    And foo is in set:
+      | "a"    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.000" |
+      |  2019-01-01T00:00:00.000   |
+
+    Then the following data should be generated:
+      | foo     |
+      | null    |
+      | "aa"   |
+      | "aaaa" |
+      | "aaaaaaaaaa" |
+      | "2019-01-01T00:00:00.000" |
+      |  2019-01-01T00:00:00.000  |
+
+
+  Scenario: Running an 'longerThan' request alongside a afterOrAt test1  constraint should be successful
```

Comment:
I know we added in "test1" when we were pairing on this but it would be good to rename this test case to something more meaningful

Reply:
Updated to meaningful test name